recipes/rmlui/4.x/patches/4.0-fix-cmake-targets-and-config.patch

```diff
@@ -0,0 +1,87 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -322,21 +322,13 @@
+ # FreeType
+ if(NOT NO_FONT_INTERFACE_DEFAULT)
+ 	find_package(Freetype REQUIRED)	
+-
+-	if(FREETYPE_FOUND)
+-		include_directories(${FREETYPE_INCLUDE_DIRS})
+-		link_directories(${FREETYPE_LINK_DIRS})
+-		list(APPEND CORE_LINK_LIBS ${FREETYPE_LIBRARY})
+-	endif()
++	list(APPEND CORE_LINK_LIBS freetype)
+ endif()
+ 
+ # Lua
+ if(BUILD_LUA_BINDINGS)
+ 	find_package(Lua REQUIRED)
```

Comment:
Looking at the recipe, I am not sure if this is case sensitive https://github.com/conan-io/conan-center-index/blob/746eeba73375f0e1815c2c27328d22dd4f1ca49d/recipes/lua/all/conanfile.py#L76

The 's none defined and the Conan default is all lowercase.

Did you test this config locally? Its not tsted by C3i

Reply:
I did test locally. Now that you mention it, I think it's likely using the built-in [FindLua](https://cmake.org/cmake/help/latest/module/FindLua.html) module. I'll change to patch to use config-only and use the lowercase version to make sure we find the Lua provided by conan.